Kleo-Network/frontend
# feat: update homepage layout and add new components

### Summary
This pull request improves the Call to Action (CTA) section on the homepage and adds a new video section to increase user engagement. The updated CTA enhances visibility and user interaction, while the new video section provides dynamic content to better communicate the project's mission.

### Changes Made
- Improved the CTA button design for better visibility and responsiveness.
- Adjusted the layout to make the CTA more prominent and engaging.
- Refined CTA copy to make it clearer and more action-oriented.
- Added a video section below the fold to enhance storytelling and highlight key features.
- Ensured the video is responsive and fits well across different screen sizes.

### Screenshots & Video
#### New:
![New-homepage](https://github.com/user-attachments/assets/b7a8d25d-be4a-4ed2-897a-c850d15ee0ff)


#### Video:
[Link to video](link-to-video)

### Related Issue
- While making these updates, I had to use older versions of certain packages due to project compatibility. This might affect future package updates or features, so a review of the current dependencies may be required to avoid potential conflicts.

### Testing Instructions
1. Navigate to the homepage.
2. Verify that the improved CTA is prominently displayed and stands out.
3. Ensure that the video is embedded correctly and plays on all devices.
4. Test both
